### PR TITLE
Fix permission on general view PluginSettings and move in template

### PR DESCRIPTION
**Pull request by Rieven**
### Changes
PluginSettingsListView had a permission for the whole view, while it must be checked in template.
Can see Plugin detail but cannot see section of plugin settings with no permission 

### Issue link
https://github.com/minvws/nl-kat-coordination/issues/1323

Closes https://github.com/minvws/nl-kat-coordination/issues/1323

### Proof
image

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/katalogus/templates/plugin_detail.html [hunk 1]
```diff
@@ -20,53 +19,49 @@ <h1>{{ plugin.name }}</h1>
...
             <div>
-              <h2>{% translate "Consumes" %}</h2>
-
-              {% if plugin.consumes %}
-                <p>
-                    {% blocktranslate trimmed with plugin_name=plugin.name %}
+                <h2>{% translate "Consumes" %}</h2>
+                {% if plugin.consumes %}
+                    <p>
+                        {% blocktranslate trimmed with plugin_name=plugin.name %}
```
Comment (Donnype) [inline rocky/katalogus/templates/plugin_detail.html:45]: Was this the work of  a linter? The changes are hard to spot right now.. Don't we enforce indentation levels?

**Review by Donnype — DISMISSED** [review]

**Review by ammar92 — DISMISSED** [review]

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* Clients can access the Boefje detail page, but cannot enable/disable them.
* Clients cannot enable/disable boefjes from the katalogus page

**Comment by ammar92** [comment]
Please fix the tests so this PR can be merged

File: rocky/katalogus/templates/plugin_detail.html (hunk 1 above)
Comment (Rieven) [reply rocky/katalogus/templates/plugin_detail.html:45]: Yes @Donnype it is indeed not nice those differences nothing to do with the PR. We still have to decide which linter we are going to use for templates, I am still using djlint

**Review by ammar92 — APPROVED** [review]

**Pull request merged**